controllers/ebpf/bpfmanager-controller.go

```diff
@@ -0,0 +1,257 @@
+package ebpf
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"github.com/netobserv/network-observability-operator/pkg/helper"
+	"github.com/sirupsen/logrus"
+
+	flowslatest "github.com/netobserv/network-observability-operator/apis/flowcollector/v1beta2"
+
+	bpfmaniov1alpha1 "github.com/bpfman/bpfman-operator/apis/v1alpha1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	netobservApp     = "netobserv"
+	netobservBCImage = "quay.io/bpfman-bytecode/netobserv:latest"
+)
+
+// BpfmanAttachNetobserv Creates BpfmanApplication object with all required ebpf hooks and attaches them using bpfman manager
+func BpfmanAttachNetobserv(ctx context.Context, client client.Client, fc *flowslatest.FlowCollector) error {
+	return bpfmanCreateNetobservApplication(ctx, client, fc, false)
+}
+
+// BpfmanDetachNetobserv Creates BpfmanApplication object with all required ebpf hooks and detaches them using bpfman manager
+func BpfmanDetachNetobserv(ctx context.Context, client client.Client, fc *flowslatest.FlowCollector) error {
+	return bpfmanCreateNetobservApplication(ctx, client, fc, true)
```

Comment:
It's kind of confusing to call `bpfmanCreateNetobservApplication` and pass in an `isDelete` flag, which when true, will do a completely different operation.  Better to have another function to delete the Netobserv application.